src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,67 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int DEFAULT_RADIUS = 10;
+    private static final int MAX_LENGTH_OF_SIDE = 10;
+    private static final Random RANDOM_FIGURE = new Random();
+    private static final Random RANDOM_SIDE = new Random();
+    private static final ColorSupplier COLOR_SUPPLIER = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        int figureIndex = RANDOM_FIGURE.nextInt(Figures.values().length);
```

Comment:
let's instead generate random index from 0 to 4 and write it in variable `randomNumber`